packages/react-scripts/template/src/components/example/HomePage.js

```diff
@@ -132,21 +134,23 @@ const HomePage = () => {
         </Cell>
 
         <Cell>
-          <RequireSignedInUser
-            Component={ArtifactsViewer}
-            fallback={
-              <Banner
-                color={colors.help.accent2}
-                message="We really want to show you some pictures of your ancestors but you must sign in first"
-              />
-            }
-          />
+          <React.Suspense fallback={<Skeleton.Image height={250} />}>
+            <RequireSignedInUser
+              Component={ArtifactsViewer}
+              fallback={
+                <Banner
+                  color={colors.help.accent2}
+                  message="We really want to show you some pictures of your ancestors but you must sign in first"
+                />
+              }
+            />
+          </React.Suspense>
```

Comment:
I think this can be just `Suspense`, though not a big deal

Reply:
K. Will fix.